src/main/java/view/OutputView.java

```diff
@@ -0,0 +1,73 @@
+package view;
+
+import constants.LottoSettingConstants;
+import data.LottoResult;
+
+import java.util.*;
+
+public class OutputView {
+    private final List<LottoResult> placements = new ArrayList<>(List.of(
+            LottoResult.FIFTH_PLACE, LottoResult.FOURTH_PLACE, LottoResult.THIRD_PLACE,
+            LottoResult.SECOND_PLACE, LottoResult.FIRST_PLACE
+    ));
+
+    public void printLottoDraftResult(List<List<Integer>> batch) {
+        System.out.println(String.format("%d개를 구매했습니다.", batch.size()));
+        for (List<Integer> single : batch) {
+            System.out.println(single.toString());
+        }
```

Comment:
```suggestion
        batch.forEach(System.out::println);
```
이처럼 메소드 참조를 쓰면 보다 간결하게 표현이 가능합니다. `List`와 같은 컬렉션을 다룰 때 스트림 API를 써보시는 것도 추천드립니다!

Reply:
코드가 매우 깔끔해졌네요! 한 수 배웁니다.